### PR TITLE
Add helper text to explore route

### LINKED ISSUE #1018: Add helper text on explore route
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**

In case events for a exist, the output is:
image

In case for some category no event is created, the output is:
image

**Expected behavior:**

When there are no events for some category, some helper text should be there.

### PR DESCRIPTION
**Pull request by ritikamotwani**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adds helper text to the explore route.

#### Changes proposed in this pull request:

image

Fixes #1018

### CODE REVIEW

File: app/templates/explore.hbs [hunk 1]
```diff
@@ -7,6 +7,8 @@
     {{#each filteredEvents as |event|}}
       {{event-card event=event isWide=true shareEvent=(action 'shareEvent')}}
       <div class="ui hidden divider"></div>
+    {{else}}
+      <div class="ui disabled header">{{t 'No Events in this Category'}}</div>
```
Comment (CosmicCoder96) [inline app/templates/explore.hbs]: @ritikamotwani please change the helper text to *No events found for selected filters *

**Comment by niranjan94** [comment]
Deployment link: https://pr-1019-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-1019-open-event-frontend-fossasia.surge.sh

**Comment by ritikamotwani** [comment]
@CosmicCoder96  @mariobehling  please review.

**Pull request merged**